### ISSUE TITLE
Unable to reliably set aria-rowcount with serverSide rowModelType

### ISSUE THREAD

**Issue by HwAtWork** [ag-grid/ag-grid#5107]
**I'm submitting a ...**  (check one with "x")
```
[x] bug report => see 'Providing a Reproducible Scenario'
[] feature request => do not use Github for feature requests, see 'Customers of AG Grid'
[] support request => see 'Requesting Community Support'
```
**Providing a Reproducible Scenario**
https://plnkr.co/edit/6jluFFTc79NGraXo?open=main.js&preview

**Current behavior**
1. AG Grid will render aria-rowcount as "-1" despite rowCount being returned in the getRows success callback; it seems to only be set for “clientSide” row model, according to gridBodyCtrl.
2. setRowCount() API is not available for non-“infinite” row models.
3. setGridAriaProperty() API works but any subsequent render resets the value, so this is not reliable.

**Expected behavior**
1. aria-rowcount set after rowCount supplied in getRows success callback.
2. aria-rowcount set after invoking setRowCount().
3. aria-rowcount persists after invoking setGridAriaProperty().

**Please tell us about your environment:**

* **AG Grid version:** X.X.X
 v27.2.0

* **Browser:**
Chrome ("Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/100.0.4896.88 Safari/537.36")

* **Language:** [all | TypeScript X.X | ES6/7 | ES5]
all

**Comment by makinggoodsoftware** [comment]
HI,

This seems like a support question and It sounds like you have an ag-grid license/evaluating. Note that github is not our main channel for support for customers.

You should contact us through zendesk, you should have access already at https://ag-grid.zendesk.com/

If that is not the case, or you are having trouble, please contact us at info@ag-grid.com where they will be able to assist you with your access.

Thanks
